payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncherContractV2.kt

```diff
@@ -46,16 +46,26 @@ class GooglePayPaymentMethodLauncherContractV2 :
         internal val config: GooglePayPaymentMethodLauncher.Config,
         internal val currencyCode: String,
         internal val amount: Long,
+        internal val label: String? = null,
         internal val transactionId: String? = null,
-        internal val injectionParams: InjectionParams? = null
+        internal val injectionParams: InjectionParams? = null,
```

Comment:
I got rid of a lot of these, I'll add this to my list to see if I can remove it here too.